website/docs/guides/provider_configuration.markdown

```diff
@@ -83,7 +87,9 @@ provider "newrelic" {
 
 -> <small>**Note:** The provider supports ***one*** region per instance of the provider.</small>
 
-
+[account ID]: https://docs.newrelic.com/docs/accounts/install-new-relic/account-setup/account-id
```

Comment:
Maybe make mention that some resource like nrql conditions can override the provider-level account ID?